tools/convert_datasets/chase_db1.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import os
+import os.path as osp
+import shutil
+import zipfile
+
+import cv2
+import mmcv
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Convert CHASE_DB1 dataset to mmsegmentation format')
+    parser.add_argument('dataset_path', help='path of CHASEDB1.zip')
+    parser.add_argument('-o', '--out_dir', help='output path')
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    dataset_path = args.dataset_path
+    if args.out_dir is None:
+        out_dir = osp.join('data', 'CHASE_DB1')
+    else:
+        out_dir = args.out_dir
+
+    print('Making directories...')
+    mmcv.mkdir_or_exist(out_dir)
+    mmcv.mkdir_or_exist(osp.join(out_dir, 'images'))
+    mmcv.mkdir_or_exist(osp.join(out_dir, 'images', 'training'))
+    mmcv.mkdir_or_exist(osp.join(out_dir, 'images', 'validation'))
+    mmcv.mkdir_or_exist(osp.join(out_dir, 'annotations'))
+    mmcv.mkdir_or_exist(osp.join(out_dir, 'annotations', 'training'))
+    mmcv.mkdir_or_exist(osp.join(out_dir, 'annotations', 'validation'))
+
+    tmp_dir = osp.join(out_dir, 'tmp')
```

Comment:
I suggest using system tmp instead of hard code 'tmp'. On the other hand, we may add an argument so that user could specify their own tmpdir.

Reply:
What is the "system tmp"?

Reply:
You may refer to the doc [here](https://docs.python.org/3/library/tempfile.html). 
You may also take a look at this [function](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/cityscapes.py#L89) for reference.